src/continuation/continuation_featurizing.jl

```diff
@@ -0,0 +1,205 @@
+export FeaturizingContinuation
+import ProgressMeter
+import Mmap
+
+struct FeaturizingContinuation{A<:AttractorsViaFeaturizing, E, M} <: BasinsFractionContinuation
+    mapper::A
+    info_extraction::E
+    par_weight::Float64
+    method::M
+    threshold::Float64
+end
+
+"""
+    FeaturizingContinuation(mapper::AttractorsViaFeaturizing; kwargs...)
+
+A method for [`basins_fractions_continuation`](@ref).
+It uses the featurizing approach discussed in [`AttractorsViaFeaturizing`](@ref)
+and hence requires an instance of that mapper as an input.
+When used in [`basins_fractions_continuation`](@ref), features are extracted
+in order to aproximate a continuation of the attractors accross the parameter space. 
+
+Different methods of continuation are implemented namely: 
+
+- Features are grouped across a parameter range. Said differently, all features
+of all initial conditions across all parameter values are put into the same "pool"
+and then grouped as dictated by the `group_config` of the mapper.
+After the grouping is finished the feature label fractions are distributed
+to each parameter value they came from.
+- The features are grouped and labeled for a single parameter slice. The continuation
+is performed by matching the labelled features from one parameter slice to the next 
+using the special distance `method`.  
+- The labelled features are grouped using an histogram binning. TODO 
```

Comment:
_How_ do I choose which method is used...?

Reply:
The method is selected in `basins_fractions_continuation`. This is a design point I am not completely satisfied with.

Reply:
Okay I have a solution, we talk in 10!

Reply:
taking notes on the code now...